### PR TITLE
cmake: enable build on riscv{64, 32}

### PR DESCRIPTION
**Pull request by Xeonacid**
tested on Arch Linux RISC-V.

### CODE REVIEW

**Review by selsta — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -325,6 +325,16 @@ if(ARCH_ID STREQUAL "s390x")
...
 endif()
 
+if(ARCH_ID STREQUAL "riscv64")
+set(RISCV   1)
+set(RISCV64 1)
+endif()
+
+if(ARCH_ID STREQUAL "riscv32")
+set(RISCV   1)
+set(RISCV32 1)
```
Comment (mj-xmr) [inline CMakeLists.txt:335]: Is this variable and its 64 variant used somewhere or did you add it for possible future use?

Comment (Xeonacid) [reply CMakeLists.txt:335]: @mj-xmr For possible future use. If you think they are now redundant, please tell me and I will remove them.

Comment (selsta) [reply CMakeLists.txt:335]: IMO it's fine to keep it

Comment (mj-xmr) [reply CMakeLists.txt:335]: I was asking mainly if I missed something in other part of the file.

It looks reasonable to have this.

**Review by mj-xmr — APPROVED** [review]

**Pull request merged**